.changeset/fast-flowers-tickle.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-kafka': patch
+'@backstage/plugin-kafka-backend': patch
```

Comment:
Given that the config for this has changed I'm wondering if this is no longer a patch change?

Reply:
Also wondering if we should elaborate on the config changes and how to migrate in this changeset too. As it's a nice way to notify people of changes to different plugins and their migration path.

Reply:
Basically something that looks like this:

```diff
-  brokers:
-    - localhost:9092
+  clusters:
+    name: prod
+    brokers:
+      - localhost:9092

Reply:
Also this change would be good to document: https://github.com/backstage/backstage/pull/4182/files#diff-b6da313ecc5cb418e89d7f58e03271d96286b41ae2e8d34347eba39a8fad1adeL80-R82

Reply:
Makes sense! updated.

Reply:
Minor is good! :) It's what it should be for breaking changes when the semver is <1.0.0